corehq/apps/hqwebapp/templates/hqwebapp/iframe_sso_login_success.html

```diff
@@ -0,0 +1,26 @@
+{% extends "hqwebapp/base.html" %}
```

Comment:
This is likely too late, and out of scope for this PR, but if SSO templates and logic are embedded inside `hqwebapp`, what is the value of a separate SSO app?

Reply:
I don't follow.

Reply:
I don't have a great reference for it, but my understanding was that django apps are meant to be 'reusable/pluggable' apps, as mentioned [here](https://django-reusable-app-docs.readthedocs.io/en/latest/). If `hqwebapp` depends on `sso` (by having functions like `iframe_sso_login_pending()`) and `sso` is dependent on `hqwebapp`, then they're really just the same app. Because we [incur a cost](https://docs.djangoproject.com/en/3.2/ref/applications/#how-applications-are-loaded) for every new app (django has to build an app registry and parse at least all the models for every app -- this is one of the reasons our _manage.py_ takes so long to load), I was asking what the upside is to keeping SSO as a separate application -- the alternative would be to just have something like a _hqwebapp.sso_ package.

Reply:
I don't see how this is an argument for SSO not needing to be a separate app when it's literally one template and I put it where the other "iframe" templates were so that it could be easily discovered / found. SSO is much more than a tiny template. This might be just too pedantic and beyond practicality at this point, no?

Reply:
This specific comment isn't blocking anything -- it's just a commentary on the overall architecture. While trying to speed up our `manage.py` process, for example, I was toggling on and off applications, and if two applications become mutually dependent on each other, that becomes impossible. 

Keep in mind that this interdependency only gets worse as time goes on and more functionality is layered on top.

We can offline about this at some later point, but I'm curious when you believe something should be its own application, and what advantages that brings over just putting it in a nested module.